### ISSUE TITLE
[Proposal]: A Walk-through Databases!

### ISSUE THREAD

**Issue by mridubhatnagar** [pydatadelhi/talks#91]
- Abstract (2-3 lines)
Explain with real-world examples SQL, NoSQL databases.

- Brief Description and Contents to be covered
1. SQL, NoSQL databases with Analogies
2. Use Cases where SQL, NoSQL can be used.
3. Types of NoSQL databases.
4. Difference between SQL, NoSQL. (With analogy explain meaning of vertical scaling, horizontal scaling)
5. Different types of NoSQL databases. 
6. Dive into MongoDB. Basic Terminology and frequently used commands.
7. PyMongo

- Pre-requisites for the talk

- Time required for the talk
50 minutes

- Link to slides[WIP]
https://docs.google.com/presentation/d/1I7h88wWudH-uIETk8LyVBZ_h7kybkq5-gCmvt0tFptQ/edit?usp=sharing 

- Will you be doing hands-on demo as well?

- Link to ipython notebook (if any)

- About yourself
I am Mridu Bhatnagar. A 2017 Computer Science and Engineering Graduate. On weekdays I contribute to building products at Goibibo.com. On weekends I love going to community meetups and give talks. :) 

- Are you comfortable if the talk is recorded and uploaded to PyData Delhi's YouTube channel?
Yes

- Any query?
No

**Comment by sanketverma1704** [comment]
Please have a look @shagunsodhani .

**Comment by shagunsodhani** [comment]
Hey @mridubhatnagar  Thank you for your proposal. Could you please share the slide deck here so that we may skim through it once :)

**Comment by mridubhatnagar** [comment]
@shagunsodhani yes sure. I am yet to prepare the slides though. By Monday I'll update here. Looking forward to the talk in the upcoming meetup :)

**Comment by sanketverma1704** [comment]
Hi @mridubhatnagar.
Any updates on slides?

**Comment by mridubhatnagar** [comment]
Most likely this week. Any meetup coming?

**Comment by sanketverma1704** [comment]
We would be having one in May.

**Comment by sanketverma1704** [comment]
Updates on slides @mridubhatnagar?

**Comment by mridubhatnagar** [comment]
Hey, haven't prepared yet. Would be starting off today.

**Comment by mridubhatnagar** [comment]
Is this topic a very trivial topic to explain as a part of the meetup?
I am confused if this is worth taking forward or not. Intent behind was to do it for beginners.
Or should this be a lightning talk?

@shagunsodhani @MSanKeys963

**Comment by shagunsodhani** [comment]
Well it can be very interesting and useful depending on how you make the slides. I would encourage you to give it a shot.
